fix.sh

```diff
@@ -35,3 +35,8 @@ xmlstarlet ed -P \
 	-i '//_:field[@name="phase"]/_:type[@name="TouchpadGesturePhase"]' -t attr -n 'c:type' -v 'gint8' \
 	Gdk-3.0.gir > tmp
 mv tmp Gdk-3.0.gir
+
+xmlstarlet ed -P \
+	-i '//*[@type="iconv_t"]' -t attr -n 'name' -v 'iconv_t' \
```

Comment:
Not `@c:type` ?

Reply:
And current command don't do anything :( Even after changing c:type

Reply:
`-i '//_:type[@c:type="iconv_t"]' -t attr -n 'name' -v 'gpointer'` produce almost same GMime-3.0.gir for me

Reply:
Thanks.
With using -L and @c:type, it now does work as expected.

Reply:
I'd assume that using gpointer here breaks any change of using a real iconv binding later on?